pkg/controller/quota/quotaforestmanager/qm_lib_backend_with_quotasubt_mgr.go

```diff
@@ -541,7 +541,14 @@ func (qm *QuotaManager) Fits(aw *arbv1.AppWrapper, awResDemands *clusterstateapi
 		qm.removeConsumer(consumerID)
 		return doesFit, preemptIds, strings.TrimSpace(allocResponse.GetMessage())
 	}
-	preemptIds = qm.getAppWrappers(allocResponse.GetPreemptedIds())
+	if len(allocResponse.GetPreemptedIds()) > 0 {
+		preemptIds = qm.getAppWrappers(allocResponse.GetPreemptedIds())
+	} else {
+		var s []string
+		s = append(s, allocResponse.GetConsumerID())
+		preemptIds = qm.getAppWrappers(s)
+	}
+
```

Comment:
@asm582,
I would code this in a slightly different way, easier to read:
```golang
    if len(allocResponse.GetPreemptedIds()) > 0 {
		preemptIds = qm.getAppWrappers(allocResponse.GetPreemptedIds())
	} else {
		preemptIds = qm.getAppWrappers([]string{allocResponse.GetConsumerID()})
	}
```

Reply:
Thanks will revert this change, the intention of this PR is to only perform config changes ie adding roles